src/style.css

```diff
@@ -0,0 +1,51 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
```

Comment:
there should be 2 font-faces with the same name, the only way to select either is by using font-weight, in this case: if `normal` – we use Avenir-Book, if `bold` – we use Avenir-Heavy
```
@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir-Book.ttf);
  font-weight: normal;
}

@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir-Heavy.ttf);
  font-weight: bold;
}
```